packages/api-explorer/src/block-types/Parameters.jsx

```diff
@@ -61,7 +61,10 @@ Parameters.propTypes = {
     data: PropTypes.shape({
       cols: PropTypes.number.isRequired,
       rows: PropTypes.number.isRequired,
-      data: PropTypes.shape({}).isRequired,
+      data: PropTypes.shape({
```

Comment:
Replace it with `PropTypes.object.isRequired`. It could be of the format of `h-*` as seen in map function of line 14 of the file